src/test/components/Home.test.js

```diff
@@ -127,7 +130,14 @@ describe('app/Home', function() {
     });
     await findByTestId('home-enable-popup-instructions');
 
-    fireFullClick(getByText('Enable Profiler Menu Button'));
+    fireFullClick(
+      getByText('Enable Firefox Profiler Menu Button', {
+        normalizer: str =>
+          // The l10n library inserts isolation character around variables. Let's
+          // remove them so that the matched string doesn't depend on it.
+          getDefaultNormalizer()(str).replace(/[\u2066-\u2069]*/g, ''),
+      })
```

Comment:
I decided to go this direction, this is described in this documentation: https://testing-library.com/docs/queries/about/#normalization

I think this is slightly better than using a regexp to add the `.` any character matcher, or adding the isolation characters with unicode, because this is an implementation detail. 

But the opposite argument works too: we can decide that using the isolation characters in the string `'Enable \u2068Firefox Profiler\u2069 Menu Button'` is important, so that we check that we have them... So tell me what you think @canova !

Also adding this normalizer everywhere is a bit tedious so in the future I'd like to see whether we could make this a default if want to go this way.

Also note that this doesn't work for `getByRole` until my PR in https://github.com/testing-library/dom-testing-library/pull/908 is merged.

Reply:
Note: if we decide that we should add the isolation characters in the string instead, then I'll remove the second commit and move it to a different PR.

Reply:
The more I think of it, the more I think adding the isolation character to the string is actually the right solution, but I'll wait until your feedback before moving forward and doing it!

Reply:
Hmm, I'm not 100% sure what to pick either. On one hand, it's good to be able to test without dealing with isolation characters. But on the other hand, it's good to be more explicit about them, so in the future we can see the differences if flow decides to put one in a different place.

If we decide to move forward with this current solution, I would prefer to have a util function for this custom normalizer. Like, a cusom `getIsolationCharactersNormalizer` in the utils file.

But I'm also closer to the idea of directly adding these characters to the string, because it's simpler and more clear.